### PR TITLE
ImageEditor: move cancel button to the side of reset button

### PR DESCRIPTION
**Pull request by rralian**
Addresses #8024. Just moving the cancel button to sit alongside the other buttons on the bottom right of the modal.

cancel button

### CODE REVIEW

File: client/post-editor/media-modal/image-editor/_style.scss [hunk 1]
```diff
@@ -93,7 +93,3 @@
 .editor-media-modal-image-editor__buttons button {
 	margin-left: 10px;
 }
-
-.editor-media-modal-image-editor__buttons-cancel {
-	float: left;
```
Comment (gwwar) [inline client/post-editor/media-modal/image-editor/_style.scss]: 😄 yay less code

**Comment by gwwar** [comment]
:shipit:

**Comment by drw158** [comment]
Since they are now all sitting alongside each other, it should be more clear what the Cancel button does. We could make the button text red maybe.

**Comment by rralian** [comment]
I think it should probably remain black so it's consistent with the cancel button from the prior image details modal which is seen immediately before the image-editing modal. And that helps to set off the "delete" action even more.

screen shot 2016-09-13 at 2 31 01 pm

**Comment by rralian** [comment]
Rebased and will merge now.

**Pull request merged**